### ISSUE TITLE
"Unknown error: Cannot read property '0' of undefined" when using console.log(browser.getRequest(0))

### ISSUE THREAD

**Issue by ksycz** [chmanie/webdriverajax#14]
Hey, I installed webdriverajax via npm (it installs "webdriverajax": "^2.1.0" by default), I have "webdriverio": "^4.8.0". 
When I try to run a code like below: 
```
it('3.1.1 - should create a new person with required data:', function () {
            PeopleCreateNewPage.open();
            PeopleCreateNewPage.newFirstName.setValue(context.new_person_first_name);
            PeopleCreateNewPage.newLastName.setValue(context.new_person_last_name);
            browser.setupInterceptor();
            browser.submitForm("#person_edit_form");
            browser.pause(5000);
            console.log(browser.getRequest(0));
```
it returns me an error: **unknown error: Cannot read property '0' of undefined** I also tried to use browser.getRequests() without any index and then I see the **Could not find request with index undefined** error, I am not sure what is an issue? Could you please take a look at that? Thank you.

**Comment by chmanie** [comment]
You're probably seeing this error because no requests were captured. Is the request maybe done inside an iframe? See https://github.com/chmanie/webdriverajax/issues/13

**Comment by chmanie** [comment]
I just published version v2.2.0 which should improve some error messages a lot. If you need further help with you specific use case please join the gitter channel.

**Comment by chmanie** [comment]
I just published version v2.2.0 which should improve some error messages a lot. If you need further help with you specific use case please join the gitter channel.
